sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/BoundingBox.java

```diff
@@ -24,7 +26,11 @@
      * @param points The list of coordinates of the Bounding box.
      */
     public BoundingBox(final List<Point> points) {
-        this.points = points;
+        if (points == null) {
+            this.points = null;
+        } else {
+            this.points = Collections.unmodifiableList(new ArrayList<>(points));
+        }
```

Comment:
Since this is now `unmodifiableList` , we may want to document this in java doc for its getter @ 
https://github.com/Azure/azure-sdk-for-java/blob/44f6e0b52fe3203626036adcd48170b715cc0640/sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/BoundingBox.java#L41